external/fileexchange/subtightplot.m

```diff
@@ -0,0 +1,67 @@
+function h=subtightplot(m,n,p,gap,marg_h,marg_w,varargin)

```

Comment:
this should come with a subtightplot.license file, since contributions on file exchange have different licenses.